helm-charts/splunk-otel-collector/values.schema.json

```diff
@@ -1409,10 +1409,6 @@
       "type": "boolean",
       "deprecated": true
     },
-    "networkExplorer": {
```

Comment:
If we remove it, the chart installation will be failing if `networkExplorer` is submitted since we don't allow unknown values. Do we want this?

Reply:
I think it should be okay to drop it now. We have had the warnings for more than 5 versions now. Do you have a specific timeline in mind?

Reply:
No specific timeline. If we think it's ready to be drop, I'm ok with that